src/main/java/com/disney/pg2k4j/CommandLineRunner.java

```diff
@@ -0,0 +1,206 @@
+package com.disney.pg2k4j;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.auth.profile.ProfileCredentialsProvider;
+import com.amazonaws.services.kinesis.producer.KinesisProducerConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+public class CommandLineRunner implements
```

Comment:
Are we all happy pulling in the additional `picocli` dependency to have this as part of the main code package?